docs/semgrep-code/notifications.md

```diff
@@ -56,15 +56,104 @@ To view, add, remove, disable, or enable your saved channels:
 2. In the **[Rule board](https://semgrep.dev/orgs/-/board)**, click the <i class="fa-solid fa-gear"></i> **gear** icon to enable or disable an integration.
     ![Screenshot of Semgrep's Rule board integration modal](/img/rule-board-integrations.png)
 
-## Adding notification channels
+## Setting up and customizing notification channels
 
 ### Slack
 
 <!-- When referring to the Slack integration, let's consistently call it the "Semgrep Slack app" (app in lower case). This is consistent with the terminology used by Slack (they refer to these integrations as apps in their product pages. It also makes sure that the user knows what "app" we're referring to.
 
 Prefer "Semgrep Slack app" over "Semgrep Slack integration" or "Slack integration" or any other variant. -->
 
-<ProcedureIntegrateSlack />
+<!-- <ProcedureIntegrateSlack /> -->
+
+The Semgrep Slack app enables Semgrep Cloud Platform to notify you of new findings after every scan. By receiving notifications within your Slack workspace, developers and security engineers are able to see findings without leaving their current environment. This can lessen the friction between detecting a finding, triaging it, and resolving it.
+
+You can select which channels in your Slack workspace will receive findings. You can also choose to receive findings only for certain repositories (projects) or certain **policies**. 
+
+**Policies **typically correspond to the different Rule board columns. For example, you can choose to receive notifications only for findings generated by rules from the Rule Board's Blocking column.
+
+:::info PREREQUISITE
+You must be a Slack **Workspace Owner** to set up the Semgrep Slack app. 
+:::
+
+#### Installing Semgrep Slack App
+
+To install the Semgrep Slack app, follow these steps:
+
+1. Sign in to your [Semgrep Cloud Platform](semgrep.dev/login) account, and then go to **Settings** > **[Integrations](https://semgrep.dev/orgs/-/settings/integrations)**.
+2. On the **[Integrations](https://semgrep.dev/orgs/-/settings/integrations)** page, click **Add Integration** (or **Setup First Integration** if this is your first integration), and then select **Slack**.
+3. Click **Allow**.
+
+#### Setting up notifications for findings in Slack
+
+To set up or subscribe to notifications for findings in your Slack workspace, perform the following steps:
+
+1. In your Slack workspace, find or create a channel for Semgrep notifications.
+2. In the selected Slack channel, enter the following slash command: `/semgrep_subscribe`.
```

Comment:
Should we be including this just as a heads up or FYI?

> If customers select >1 filter, i.e. project, policy, product; it's more like an **AND** instead of **OR**. Which means, a user will only receive notifications when all of those conditions are met.

Reply:
Thanks for this heads up! I'll add a section in troubleshooting "Not receiving any findings".